_output.yml

```diff
@@ -1,3 +1,12 @@
+bookdown::bs4_book:
+  theme:
+    primary: "#982a31"
```

Comment:
I don't know if we need to formally record this anywhere but these colours were produced by taking O'Reilly red (#BF353F) and setting chroma to 80 and lightning to 35 for the primary colour, and chroma to 5 and lightness to 14 for the text colour.